ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.10__okd.yaml

```diff
@@ -57,8 +57,11 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
+  commands: make community
   steps:
     cluster_profile: aws
+    env:
+      COMMUNITY: true
```

Comment:
You'll also need to add this env var to the [config of the referenced test](https://github.com/openshift/release/blob/master/ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml#L9)